docs/en/observability/synthetics-troubleshooting.asciidoc

```diff
@@ -100,9 +100,37 @@ If the duration is _less than_ the scheduled frequency or the suggestion above d
 
 To fix this issue, you can either:
 
-* Increase the number of concurrent browser monitors allowed (as described in <<synthetics-private-location-scaling,Scaling Private Locations>>), paying attention to the scaling and hardware requirements documented).
+* Increase the number of concurrent browser monitors allowed (as described in <<synthetics-private-location-scaling,Scaling Private Locations>>), paying attention to the scaling and hardware requirements documented.
 * Create multiple {private-location}s and spread your browser monitors across them more evenly (effectively horizontally scaling your {private-location}s).
 
+[discrete]
+[[synthetics-troubleshooting-no-locations]]
+== No locations are available
```

Comment:
Thinking about this, we should probably clarify this is only for ESS users, too.  People will read these docs for self-managed too, and they won't see or be able to use, public locations.